client/src/components/Inventory/Bikes.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { Fragment, useState } from 'react';
+import { Fragment } from 'react';
```

Comment:
This might not be related to this PR but do we really need the Fragment? can't we just add the key to the row? Is there any advantage of putting the key in Fragment?

Reply:
You're absolutely right and I noticed it, but I didn't want to modify it LOL